lib/rubocop/cop/rails/time_zone.rb

```diff
@@ -0,0 +1,115 @@
+# encoding: utf-8
+
+module RuboCop
+  module Cop
+    module Rails
+      # This cop checks for the use of Time.now instead of Time.zone.now etc
+      # Built on top of this article http://danilenko.org/2012/7/6/rails_timezones/
+      class TimeZone < Cop
+        include ConfigurableEnforcedStyle
+
+        MSG = 'Do not use `%s` without zone. Use `%s` instead.'
+
+        MSG_SEND =  'Do not use `%s` on Date objects,' \
+                    'because it knows nothing about time zone in use.'
+
+        TIMECLASS = [:Time, :DateTime]
+
+        DANGER_METHODS = [:now, :local, :new, :strptime, :parse, :at]
+
+        BAD_DAYS = [:today, :current, :yesterday, :tomorrow]
+
+        def on_const(node)
+          _, klass = *node.children
```

Comment:
We prefer to avoid the use of `_` in favour of `_` prefixed names - e.g. `_module`. Makes it easier to understand what you're destructuring. For the same reason we avoid the use of `node.children` (instead we usually use destructuring).